addons/mass_mailing/static/src/less/themes/theme_default.less

```diff
@@ -51,10 +52,9 @@
 .o_layout {
     overflow: hidden;
     box-sizing: content-box;
-    width: 94%;
+    width: 100%;
     min-height: 100%;
     background-color: @o_mm_def_color_alpha;
-    padding: 24px 3%;
```

Comment:
In fact, the top and bottom paddings are needed for the design of the other themes, but I will change that myself and merge :)